globals.go

```diff
@@ -359,10 +359,11 @@ const (
 	// to tear down a task group. This is set one minute longer than the agent's maxTeardownGroupTimeout.
 	MaxTeardownGroupThreshold = 4 * time.Minute
 
-	SaveGenerateTasksError     = "error saving config in `generate.tasks`"
-	TasksAlreadyGeneratedError = "generator already ran and generated tasks"
-	KeyTooLargeToIndexError    = "key too large to index"
-	InvalidDivideInputError    = "$divide only supports numeric types"
+	SaveGenerateTasksError          = "error saving config in `generate.tasks`"
+	TasksAlreadyGeneratedError      = "generator already ran and generated tasks"
+	KeyTooLargeToIndexError         = "key too large to index"
+	InvalidDivideInputError         = "$divide only supports numeric types"
+	FetchingTaskDataUnfinishedError = "fetching task data not finished"
```

Comment:
If we're going to rely on this error string to determine that the task data didn't finish fetching in time, can we also use this string constant in [the source error](https://github.com/evergreen-ci/evergreen/blob/ebf8cc72517c84a164df1a3a2067cf5987dad1e8/model/host/hostutil.go#L958)?